inspector/src/components/actionTabs/tabs/performanceViewer/performanceViewerComponent.tsx

```diff
@@ -20,11 +20,10 @@ interface IPerformanceViewerComponentProps {
     scene: Scene;
 }
 
-// aribitrary window size
+// arbitrary window size
 const initialWindowSize = { width: 1024, height: 512 };
 
-// Note this should be false when committed until the feature is fully working.
-const isEnabled = false;
+const isEnabled = true;

```

Comment:
should we not remove the flag altogether ??? now that it is live

Reply:
yes, that makes sense

Reply:
Agreed

Reply:
Removed the flag :D